docs/topic/secrets.md

```diff
@@ -0,0 +1,40 @@
+# Secrets and private keys
+
+## Where are secrets stored
+
+Most secrets are stored in this location:
+
+https://github.com/2i2c-org/pilot-hubs/blob/master/config/secrets.yaml
```

Comment:
Well, actually, most secrets live here instead :wink: : https://github.com/2i2c-org/pilot-hubs/tree/master/secrets

Reply:
Ha good point - do we have it documented when to use one folder and not another? If not we can add that here

Reply:
> do we have it documented when to use one folder and not another?

Not sure if we have an explicit reference...

Reply:
If we don't have an explicit reference, could you eyeball an informal guideline to use? Like are there certain kinds of secrets that we store in one folder vs. another?

Reply:
Sure, 

> https://github.com/2i2c-org/pilot-hubs/blob/master/config/secrets.yaml

The above one deals auth0 credentials and a general proxy secret for our hubs. More info here: https://github.com/2i2c-org/pilot-hubs/blob/master/deployer/__main__.py#L57-L83

Whereas 

> https://github.com/2i2c-org/pilot-hubs/tree/master/secrets

deals with cluster authentication for all the cloud providers we support. More info here: https://github.com/2i2c-org/pilot-hubs/blob/master/deployer/hub.py#L39-L46 and https://github.com/2i2c-org/pilot-hubs/blob/master/deployer/hub.py#L126-L216.

Reply:
OK I'll add those into this PR.

Though, I was looking for more like a _reason_ that we use two different places. Like "We store authentication credentials in XXX, because of YYY, and we store cluster authentication credentials in ZZZ, because of WWW"

Reply:
Oh... sorry, I should have been more clear...
* General secrets (used by all of our clusters): https://github.com/2i2c-org/pilot-hubs/blob/master/config/secrets.yaml
* Cluster specific secrets: https://github.com/2i2c-org/pilot-hubs/tree/master/secrets